### PR TITLE
Typos and misspellings in synset examples

### PR DESCRIPTION
**Pull request by 1313ou**
After applying a deep correction-model (ai-forever/sage) to raise suspicious cases (including punctuation, unknown terms, archaic words, ...), I reviewed the long list (about 10%) and hand-corrected to a short list of unambiguous cases (this PR).

I chose  a deep model to handle such cases as "The glass was opacified more greater privacy" where a word-by-word spellchecker fails to see the issue arising from assembling individually valid words.

This deals with examples only, to the exclusion of definitions. Punctuation, capitalization and other issues (like the use of the infamous '--') have been left aside for now. Hopefully these will also be handled at a later stage.

The 'hand' review and correction mean some misspellings may have sneaked through. Inversely I may have introduced false positives. So external reviewing is required.

### CODE REVIEW

File: src/yaml/adj.all.yaml [hunk 1]
```diff
@@ -53500,7 +53500,8 @@
   example:
   - had a preference for circumlocutious (or circumlocutory) rather than forthright
     expression
-  - source: T.S.Eliot; (`ambagious' is archaic)
+  - (`ambagious' is archaic)
```
Comment (jmccrae) [inline src/yaml/adj.all.yaml:53503]: The source should be "T.S.Eliot"... the text "(`ambagious' is arachaic)" should probably be removed

File: src/yaml/entries-b.yaml [hunk 2]
```diff
@@ -6232,10 +6232,10 @@ Bletilla striata:
     sense:
     - id: 'bletilla_striata%1:20:00::'
       synset: 12067284-n
-Bleu Vegetal Pour Les Veines:
+Bleu Végétal Pour Les Veines:
```
Comment (jmccrae) [inline src/yaml/entries-b.yaml:6235]: In this case, we generally add an entry for both with and without accents

File: src/yaml/noun.communication.yaml [hunk 3]
```diff
@@ -61682,7 +61682,7 @@
   - An accent on the first syllable of a word.
   example:
   - The fronting took place more slowly in American English and so Modern American
-    English has forms like elo'ngate, imprégnate and remonstrate where British English
+    English has forms like eˈlongate, imˈpregnate and reˈmonstrate where British English
```
Comment (jmccrae) [inline src/yaml/noun.communication.yaml:61685]: I don't think this is correct. Original source is here:
https://books.google.ie/books?id=0NOidwn8Mf0C&pg=PA202&lpg=PA202&dq=%22The+fronting+took+place+more+slowly+in+American+English+and+so+Modern+American%22&source=bl&ots=vm70LX5UJJ&sig=ACfU3U0xkns5-6hBFGUZvzwbIeVSCqQoHg&hl=en&sa=X&ved=2ahUKEwiR-5rGxcGGAxVraEEAHZFnEqkQ6AF6BAgJEAM#v=onepage&q=%22The%20fronting%20took%20place%20more%20slowly%20in%20American%20English%20and%20so%20Modern%20American%22&f=false

**Review by jmccrae — CHANGES_REQUESTED** [review]

File: src/yaml/noun.communication.yaml (hunk 3 above)
Comment (1313ou) [reply src/yaml/noun.communication.yaml:61685]: Link is dead for me. As I understand it the example was to mean that elongate, impregnate, remonstrate have stress on the 2nd syllable in US English, first in British English. IPA uses accents BEFORE the stressed syllable. 'imprégnate', à-la-French, is clearly unacceptable. What do you suggest ?

File: src/yaml/entries-b.yaml (hunk 2 above)
Comment (1313ou) [reply src/yaml/entries-b.yaml:6235]: What was unacceptable was the hybrid 'Végetal'. Either both 'e's are accented or none. While I was at it I thought it better to correct it the French way.

File: src/yaml/adj.all.yaml (hunk 1 above)
Comment (1313ou) [reply src/yaml/adj.all.yaml:53503]: I agree. I thought, the "(`ambagious' is archaic)" part should be preserved as a usage note (more than an example) but is has nothing to do the TS Eliot's quotation. So I moved it out. What is the usage attribute to put it in?-

Comment (jmccrae) [reply src/yaml/adj.all.yaml]: The best fix would be to add the link `exemplifies archaism%1:10:00::`.

File: src/yaml/noun.communication.yaml (hunk 3 above)
Comment (jmccrae) [reply src/yaml/noun.communication.yaml:61685]: I would suggest the following
```yaml
text: The fronting took place more slowly in American English and so Modern American
    English has forms like elóngate, imprégnate and remónstrate where British English
    usually has initial stress.
source: English Phonetics and Phonology for Spanish Students - Brian Mott
```

**Comment by 1313ou** [comment]
[See Merriam Webster on how to use dashes](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use)

**Review by jmccrae — APPROVED** [review]

**Pull request merged**